std/os/windows/bits.zig

```diff
@@ -6,6 +6,7 @@ const assert = std.debug.assert;
 const maxInt = std.math.maxInt;
 
 pub const ERROR = @import("error.zig");
+pub const LANG = @import("lang.zig");
```

Comment:
Since this is neatly prefixing the LANG_ constants with this LANG namespace, want to chop off the LANG prefix in the lang.zig file? This pattern is established by ERROR above.

Reply:
Would it make sense for LANG to be an enum?